doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+
+Alternatively, you can use an external debug probe, such as the nRF5340 DK, or any J-Link device supporting ARM Cortex-M33 to program firmware on a Thingy:53.
+
+See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to program a Thingy:53 using nRF Connect Programmer or the nRF Programmer for Android or iOS.
+
+.. _thingy53_building_pgming:
+
+Building and programming from the source code
+*********************************************
+
+You can also program the Thingy:53 by using the images obtained by building the code in an |NCS| environment.
+
+To set up your system to be able to build a compatible firmware image, follow the :ref:`getting_started` guide for |NCS|.
+The build targets of interest for Thingy:53 in |NCS| are as follows:
+
++--------------------------------+----------------------------------------------------------+
+|Component                       |  Build target                                            |
++================================+==========================================================+
+| nRF5340 SoC - Application core |``thingy53_nrf5340_cpuapp`` for the secure domain         |
+|                                |                                                          |
+|                                |``thingy53_nrf5340_cpuapp_ns`` for the non-secure version |
++--------------------------------+----------------------------------------------------------+
+| nRF5340 SoC - Network core     |``thingy91_nrf52840``                                     |
++--------------------------------+----------------------------------------------------------+
+
+The table below shows the different types of build files that are generated and the different scenarios in which they are used:
+
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| File                        | File format                          | Programming scenario                                               |
++=============================+======================================+====================================================================+
+| :file:`merged.hex`          | Full image, HEX format               | Using an external debug probe and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_signed.hex`      | MCUboot compatible image, HEX format | Using the built-in bootloader and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_update.bin`      | MCUboot compatible image, HEX format | Using the built-in bootloader and the mcumgr command line tool     |
```

Comment:
there is also file for network core (`net_core_app_update.bin`)- if one is to update the board manually both images will have to be used.

This is important as we already know that our samples does not use compatible network core FW.

Reply:
Correct. I forgot to add the net_core_app_update.bin to the list. The discription can be identical to the app_update_bin image.